apps/lockfile-explorer/src/start.ts

```diff
@@ -103,7 +105,17 @@ function startApp(debugMode: boolean): void {
 
   app.get('/api/lockfile', async (req: express.Request, res: express.Response) => {
     const pnpmLockfileText: string = await FileSystem.readFileAsync(appState.pnpmLockfileLocation);
-    const doc = yaml.load(pnpmLockfileText);
+    const doc = yaml.load(pnpmLockfileText) as Lockfile;
+    const { packages, lockfileVersion } = doc;
```

Comment:
The `lockfileVersion` is a pretty important field here.  We should probably parse it into a `number` variable (or major/minor variables?) that can more easily be compared. And ideally we should detect format versions that are unsupported (because they are too old or too new) and in that case, display a friendly-looking error message to the end user.

Reply:
I believe lockfile-explorer tool only support v5/v6 for now. For other versions, I have implemented checks and throw an error.